docs/modules/getting_started/examples/code/kafka.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: kafka.stackable.tech/v1alpha1
+kind: KafkaCluster
+metadata:
+  name: simple-kafka
+spec:
+  version: 3.2.0-stackable0.1.0
+  zookeeperConfigMapName: simple-kafka-znode
+  config:
+    tls: null
+  brokers:
+    config:
+      resources:
+        storage:
+          logDirs:
+            capacity: '2Gi'
```

Comment:
I suggest we leave out all this resource stuff to keep it simple and instead demo this in more detail in a different document.

Reply:
done